chromium_src/components/content_settings/core/common/features.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_CONTENT_SETTINGS_CORE_COMMON_FEATURES_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_CONTENT_SETTINGS_CORE_COMMON_FEATURES_H_
+
+#include "../../../../../../components/content_settings/core/common/features.h"
+
+namespace content_settings {
+
+COMPONENT_EXPORT(CONTENT_SETTINGS_FEATURES)
+extern const base::Feature kBraveEphemeralStorage;
```

Comment:
this belongs in component/ephemeral_storage, not content_settings

Reply:
It's here because it is used by `chromium_src/components/content_settings/core/common/cookie_settings_base.cc ` which is compiled into the network service. This means that making it part of `components/ephemeral_storage` isn't possible, or at least I couldn't figure out a way to not create a layering violation. If we can do away with the changes to `cookie_settings_base.cc` this can move to `components/ephemeral_storage` though.